website/docs/components/form/checkbox/partials/code/how-to-use.md

```diff
@@ -50,7 +43,7 @@ The `@name` argument offers an easy way to provide the same name for all the Che
 To better fit your spacing requirements, choose between two different layout orientations: `vertical` or `horizontal`.
 
 ```handlebars
-<Hds::Form::Checkbox::Group as |G|>
+<Hds::Form::Checkbox::Group @name="datacenter" as |G|>
```

Comment:
Suggestion: consider prefixing all the `@name`s with `demo-` so it's more obvious to our consumers that they have to update that value according to their context.

```suggestion
<Hds::Form::Checkbox::Group @name="demo-datacenter" as |G|>
```